src/api/fareContracts.ts

```diff
@@ -11,9 +12,16 @@ export async function list(): Promise<ListTicketsResponse> {
   return response.data;
 }
 
+export async function listTypes(): Promise<FareContractType[]> {
```

Comment:
Should we call these `ProductTypes` instead? As far as I know, a `FareContract` can contain multiple products? This might be a good discussion point wrt domain modeling, of course.

Reply:
The name was based on the json file, which was named `fare-contract-types.json`. The actual endpoint we now use is named `preassigned-fare-products`. Is naming the type `PreassignedFareProduct` ok?

Reply:
Renamed to `PreassignedFareProduct` for now.